Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -4053,7 +4052,7 @@ final class BuildPlanTests: XCTestCase {
             XCTAssertMatch(contents, .contains("""
               "C.rary-debug.a":
                 tool: shell
-                inputs: ["\(buildPath.appending(components: "rary.build", "rary.swift.o").escapedPathString())","\(buildPath.appending(components: "rary.build", "rary.swiftmodule.o").escapedPathString())"]
+                inputs: ["\(buildPath.appending(components: "rary.build", "rary.swift.o").escapedPathString())","\(buildPath.appending(components: "rary.build", "rary.swiftmodule.o").escapedPathString())","\(buildPath.appending(components: "rary.product", "Objects.LinkFileList").escapedPathString())"]
```

Comment:
nit: format long lines

Reply:
`swiftformat` results seem pretty bad tbh, but that's what we signed up for, I guess.